app/src/main/java/com/cmm/wdflowers/ui/orders/details/OrderDetailsFragment.kt

```diff
@@ -0,0 +1,46 @@
+package com.cmm.wdflowers.ui.orders.details
+
+import android.os.Bundle
+import android.view.View
+import androidx.lifecycle.Observer
+import com.cmm.wdflowers.R
+import com.cmm.wdflowers.databinding.FragmentOrderDetailsBinding
+import com.cmm.wdflowers.ui.base.BaseBindingFragment
+import com.cmm.wdflowers.ui.orders.OrdersViewModel
+import com.cmm.wdflowers.ui.orders.model.OrderDetailsUiModel
+import com.cmm.wdflowers.ui.orders.model.OrderUiModel
+import org.koin.android.viewmodel.ext.android.viewModel
+
+const val ORDER_ID = "order_id"
+
+class OrderDetailsFragment : BaseBindingFragment<FragmentOrderDetailsBinding>() {
+
+    override fun getLayoutId() = R.layout.fragment_order_details
+
+    private val ordersViewModel: OrdersViewModel by viewModel()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        arguments?.let {
```

Comment:
We usually use [safe args](https://developer.android.com/guide/navigation/navigation-pass-data#Safe-args) to pass the data instead of bundle. It spares you of the hardcoded key

Reply:
It's actually the first time I've used the navigation component and for now, for the simplicity, chose to use the old method. But yes, safe args is the way to go for a project. Thanks!